Mathlib/Analysis/Analytic/Composition.lean

```diff
@@ -235,7 +235,7 @@ theorem comp_coeff_zero (q : FormalMultilinearSeries 𝕜 F G) (p : FormalMultil
   have : {c} = (Finset.univ : Finset (Composition 0)) := by
     apply Finset.eq_of_subset_of_card_le <;> simp [Finset.card_univ, composition_card 0]
   rw [← this, Finset.sum_singleton, compAlongComposition_apply]
-  symm; congr! -- Porting note: needed the stronger `congr!`!
+  symm; congr!
```

Comment:
Reasoning: The two sides are not obviously equal to me at all, so I think `congr!` makes sense.